### PR TITLE
Add virtualenv to pure theme

### PR DESCRIPTION
**Pull request by pix0r**

### CODE REVIEW

File: themes/pure/pure.theme.bash [hunk 1]
```diff
@@ -31,11 +31,18 @@ pure_prompt() {
     ps_root_mark="${red} # ${normal}"
     ps_path="${yellow}\w${normal}";
 
+	# Virtualenv
```
Comment (nwinkler) [inline themes/pure/pure.theme.bash]: The indentation looks wrong - it does not seem to match the rest of the file. Can you please check to make sure you're using the same 4 spaces for indentation, as in the rest of the file?

File: themes/pure/pure.theme.bash [hunk 2]
```diff
@@ -31,11 +31,18 @@ pure_prompt() {
...
+	if [[ $VIRTUAL_ENV != "" ]]; then
+		venv="(${VIRTUAL_ENV##*/})"
+	else
```
Comment (nwinkler) [inline themes/pure/pure.theme.bash]: You could avoid the `else` block if you initialized the `venv` variable before the `if`:

```bash
local venv=""
```

**Comment by pix0r** [comment]
Thanks @nwinkler, appreciate the feedback. Clearly I had not spent much time on this. Some further digging revealed that there is already a `virtualenv_prompt` available in the base theme, so I changed to use that instead.

**Comment by nwinkler** [comment]
Great - thanks!

**Pull request merged**